### DISCUSSION TITLE
Ability to install via Enterprise device management. Including sudo components

### DISCUSSION THREAD

**Discussion by SamStenton** [rancher-sandbox/rancher-desktop#2476] [Ideas]
Love rancher desktop, and would like to deploy it to our organisation to allow developers to make use of local kubernetes. In my testing it seems to be far more stable than Docker Desktop when developing applications on Mac M1 (ARM). 

Like some other discussions on here, I'm looking to install for users who do not have sudo/admin privileges. I can do this in docker desktop when installing by using a `--install-privileged-components` flag in my install script. I have seen I can install without sudo and make it usable for users but I'd like to give them the ability to open up services without port forwarding. 

Does Rancher Desktop have similar functionality? If not, would you consider making this an option.

**Comment by jandubois** [comment]
We do have a plan to replace all usage of `sudo` with a privileged helper process, but there is no concrete timeline for it yet. Hopefully still later this year.
